main.go

```diff
@@ -8,10 +8,11 @@ import (
 	"github.com/hisshoes/crypto-rebalancer/pkg/http/rest"
 	"github.com/hisshoes/crypto-rebalancer/pkg/portfolio"
 	"github.com/hisshoes/crypto-rebalancer/pkg/storage/memory"
+	"github.com/hisshoes/crypto-rebalancer/pkg/storage/mongodb"
 )
 
 func main() {
-	dbType := "memory"
+	dbType := "mongodb"
```

Comment:
This could be supplied as an environment variable (https://github.com/kelseyhightower/envconfig) or as a flag through the go flag support, if you really wanted to.